requirements.txt

```diff
@@ -0,0 +1,7 @@
+jupyterlab
+notebook
+asyncpg
+llama-index
+psycopg2
```

Comment:
@leejaeka :

we're using psycopg3 in ailab-db : https://www.psycopg.org/ 

https://github.com/ai-cfia/ailab-db/blob/7db06b4a040f29ff11a65f6ab1d53b0c97b878ad/requirements.txt#L20

using psycopg2 should be deprecated as it is an older version.

Reply:
psycopg2 was used to connect to postgresql. If psycopg3 does the same job, let us use the updated psycopg3

Reply:
Done